components/gfx/text/util.rs

```diff
@@ -149,5 +149,6 @@ pub fn is_cjk(codepoint: char) -> bool {
     }
 
     // https://en.wikipedia.org/wiki/Plane_(Unicode)#Supplementary_Ideographic_Plane
-    unicode_plane(codepoint) == 2
+    // https://en.wikipedia.org/wiki/Plane_(Unicode)#Tertiary_Ideographic_Plane
```

Comment:
can we keep both wikipedia links? one for plane 2, one for plane 3

Reply:
I have kept both the links for plane 2 (line 151) and plane 3 (line 152).

Reply:
oops, sorry, i misread the diff :sweat_smile: